system_test/helpers/aest_nodes.erl

```diff
@@ -169,6 +171,10 @@
 
 -export_type([test_ctx/0]).
 
+%% Use this gas price in tests
+gas_price() ->
+    1000000000.
```

Comment:
```suggestion
    1000000.
```

Reply:
No. Letting the system test use realistic values is a good idea.